Mathlib/Algebra/Group/Subgroup/Basic.lean

```diff
@@ -1751,6 +1756,13 @@ theorem comap_injective_isCommutative {f : G' →* G} (hf : Injective f) [H.IsCo
 instance subgroupOf_isCommutative [H.IsCommutative] : (H.subgroupOf K).IsCommutative :=
   H.comap_injective_isCommutative Subtype.coe_injective
 
+@[to_additive]
+lemma mul_comm_of_mem_isCommutative [H.IsCommutative] {a b : G} (ha : a ∈ H) (hb : b ∈ H) :
+    a * b = b * a := by
+  have := mul_comm (⟨a, ha⟩ : H) (⟨b, hb⟩ : H)
+  simp only [Submonoid.mk_mul_mk, Subtype.mk.injEq] at this
+  exact this
```

Comment:
```suggestion
  have := mul_comm (⟨a, ha⟩ : H) (⟨b, hb⟩ : H)
  simpa only [Submonoid.mk_mul_mk, Subtype.mk.injEq] using this
```
Does that work? Maybe you can even do `simpa only ... using mul_comm ...`?

Reply:
Thank you for the suggestion.

```suggestion
  simpa only [Submonoid.mk_mul_mk, Subtype.mk.injEq] using mul_comm (⟨a, ha⟩ : H) (⟨b, hb⟩ : H)
```

Yes, it does work (at least on my local environment).